test/sequential/test-vm-timeout-rethrow.js

```diff
@@ -6,14 +6,10 @@ const spawn = require('child_process').spawn;
 
 if (process.argv[2] === 'child') {
   const code = 'let j = 0;\n' +
```

Comment:
Can you reduce the whole block to `while(true);` while you're at it? I.e. no need for declaring `j` either.